src/winstanley/WdlBraceMatcher.scala

```diff
@@ -0,0 +1,20 @@
+package winstanley
+
+import com.intellij.lang.{BracePair, PairedBraceMatcher}
+import com.intellij.psi.PsiFile
+import com.intellij.psi.tree.IElementType
+import winstanley.psi.WdlTypes
+
+class WdlBraceMatcher extends PairedBraceMatcher {
```

Comment:
consider a javadoc `@see` link for official doc?  Not really sure what's going on here.